### PR TITLE
#39 Add combinable task

### PR DESCRIPTION
**Pull request by JacekDubikowski**
Closing #39

### CODE REVIEW

File: ddd-koans/Value Object Pt. 2 & Entity/Combinable/task-info.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+type: edu
+files:
+- name: src/Main.kt
+  visible: true
+  placeholders:
+  - offset: 227
+    length: 127
+    placeholder_text: TODO("How should the length of notes be calculated?")
```
Comment (jakubiec) [inline ddd-koans/Value Object Pt. 2 & Entity/Combinable/task-info.yaml:8]: doesn't look like a part of this task

Comment (jakubiec) [reply ddd-koans/Value Object Pt. 2 & Entity/Combinable/task-info.yaml:8]: or maybe this is becuase of dependency :thinking:

**Review by jakubiec — APPROVED** [review]
Looks good :+1:

File: ddd-koans/Value Object Pt. 2 & Entity/Combinable/src/Main.kt [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import NoteValue.*
+import java.math.BigDecimal
+import java.math.MathContext
+import java.time.Duration
```
Comment (jakubiec) [inline ddd-koans/Value Object Pt. 2 & Entity/Combinable/src/Main.kt]: unused import

File: ddd-koans/Value Object Pt. 2 & Entity/Combinable/task-info.yaml (hunk 1 above)
Comment (JacekDubikowski) [reply ddd-koans/Value Object Pt. 2 & Entity/Combinable/task-info.yaml:8]: Yeah, this is due to being dependency.

**Pull request merged**